🤖 Auto-merge documentation from branches

buger opened the pull request:
### **User description**
## 📚 Documentation Merge Summary

This PR contains automatically merged documentation from multiple branches.

**Triggered by:**
- **Branch:** main
- **Commit:** 9aa233e4c6eaa55496df6c2a7f39825ccf7c1cff
- **PR:** #876 - [TT-15909] Docs update for Graphql Playground.mdx

**Deployment Details:**
- **Generated from:** `branches-config.json`  
- **Run ID:** 19405492881
- **Subfolder:** `(root deployment)`
- **Timestamp:** $(date)

### Changes Include:
- ✅ Merged documentation from multiple branches
- ✅ Generated unified `docs.json` configuration
- ✅ Updated assets and content structure
- ✅ Cleaned up outdated version folders

---

🚦 **This PR will be processed by merge queue to ensure proper validation and ordering.**

Previous deployment PRs have been automatically closed to prevent conflicts.

[TT-15909]: https://tyktech.atlassian.net/browse/TT-15909?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ


___

### **PR Type**
Documentation


___

### **Description**
- Add GraphQL docs support in portal

- Explain GraphQL Playground usage

- Update docs-only product guidance

- Clarify REST vs GraphQL upload formats


___

### Diagram Walkthrough


```mermaid
flowchart LR
  docs["Portal docs: API Products"] -- "add GraphQL support notes" --> spec["API specs: OpenAPI + GraphQL SDL"]
  spec -- "interactive usage" --> playground["Built-in GraphQL Playground"]
  publish["Publish API Catalog guide"] -- "REST & GraphQL formats" --> spec
  access["Request Access guide"] -- "testing guidance" --> rest["REST via cURL/Postman"]
  access -- "interactive testing" --> playground
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>api-products.mdx</strong><dd><code>Add GraphQL SDL and Playground guidance</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/portal/api-products.mdx

<ul><li>Introduce GraphQL support alongside OpenAPI.<br> <li> Detail upload formats for REST/GraphQL specs.<br> <li> Describe Live Testing incl. GraphQL Playground.<br> <li> Extend docs-only product steps and best practices.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/958/files#diff-d04bf8b9e1339a1916636102e8e85f96ee979708cacdcc22acba78bda5170be2">+15/-6</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>publish-api-catalog.mdx</strong><dd><code>Document REST and GraphQL spec uploads</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/portal/publish-api-catalog.mdx

<ul><li>Note selecting REST or GraphQL APIs.<br> <li> Add upload instructions for OpenAPI and GraphQL SDL.<br> <li> Minor wording/grammar improvements.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/958/files#diff-368f96e99857bc67c320d97b40b435d7f492f71d5271d5eb9a407acd17c8a3af">+5/-3</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>request-access.mdx</strong><dd><code>Add GraphQL Playground testing instructions</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

nightly/portal/request-access.mdx

<ul><li>Add REST testing label and example context.<br> <li> Introduce GraphQL Playground testing note.<br> <li> Clarify testing flow by API type.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/958/files#diff-a188c18dc24b2a415d4307e13ac51b6d33b1b18c45922f8442b28142de7c756c">+5/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

marksou commented:
@l3ender Already addressed in this PR - https://github.com/TykTechnologies/tyk-docs/pull/949

buger committed:
🤖 Auto-merge documentation from branches

Generated from branches-config.json at

- Merged documentation from multiple branches
- Generated unified docs.json
- Updated assets and content structure
